logs_sdk/lib/opentelemetry/sdk/logs/logger_provider.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Logs
+      # The SDK implementation of OpenTelemetry::Logs::LoggerProvider.
+      class LoggerProvider < OpenTelemetry::Logs::LoggerProvider
+        attr_reader :resource, :log_record_processors
+
+        UNEXPECTED_ERROR_MESSAGE = 'unexpected error in ' \
+          'OpenTelemetry::SDK::Logs::LoggerProvider#%s'
```

Comment:
I'd expect this constant to be `private`
```suggestion
        UNEXPECTED_ERROR_MESSAGE = 'unexpected error in ' \
          'OpenTelemetry::SDK::Logs::LoggerProvider#%s'

        private_constant :UNEXPECTED_ERROR_MESSAGE

```